modules/quickstart/pages/quickstart-intro.adoc

```diff
@@ -7,34 +7,34 @@
 ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 
 [[quick-start-intro]]
-This {sdk-long-name} provides tools, sample code, and documentation to help you create programs to run on the {IC}.
-The _Quick Start_ tutorial assumes that you are installing the {sdk-short-name} for the first time and illustrates how to create, build, and deploy your first application. 
+This {sdk-long-name} provides tools, sample code, and documentation to help you create canister smart contract dapps to run on the {IC}.
+The _Quick Start_ tutorial assumes that you are installing the {sdk-short-name} for the first time and illustrates how to create, build, and deploy your first dapp. 
 
 To keep the instructions simple and focused on the task at hand, the _Quick start_ is split into two scenarios:
 
-* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample application with {IC} processes that run _locally to simulate connecting to the network_.
+* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample dapp with processes that in a _local canister execution environment_.
 
-* In the link:network-quickstart{outfilesuffix}[Network deployment] scenario, you create the sample application locally but _connect to the {IC} running remotely_ on computers in an external data center.
+* In the link:network-quickstart{outfilesuffix}[On-chain deployment] scenario, you create the sample dapp locally but _connect to the {IC} blockchain_ running on node machines in independent data centers and deploy the dapp there.
```

Comment:
Based on [this](https://dfinity.slack.com/archives/C02FV87QYKS/p1632389444118200), I started using "{IC} blockchain mainnet" when is about deploying canisters.

Reply:
Thanks! I'll use it with "mainnet" once and then {platform}, so it should be easy to replace if we ever change our marketing strategy again :)